deployment/aws/installer/roles/install-bk/files/install-bk.sh

```diff
@@ -18,7 +18,7 @@ PRAVEGA_CLUSTER_NAME=${PRAVEGA_CLUSTER_NAME:-"pravega-cluster"}
 BK_CLUSTER_NAME=${BK_CLUSTER_NAME:-"bookkeeper"}
 BK_AUTORECOVERY=${BK_AUTORECOVERY:-"false"}
 
-BK_LEDGERS_PATH="/${PRAVEGA_PATH}/${PRAVEGA_CLUSTER_NAME}/${BK_CLUSTER_NAME}/ledgers"
+BK_LEDGERS_PATH="/${PRAVEGA_PATH}/${BK_CLUSTER_NAME}/ledgers"
```

Comment:
Why is this change needed? PRAVEGA_CLUSTER_NAME is passed as environment variable to Bookkeeper Cluster so bk-operator should be able to access and use this name.
This is a breaking change and is going to break all existing pravega installations on upgrade.